ethereum-types/Cargo.toml

```diff
@@ -1,6 +1,6 @@
 [package]
 name = "ethereum-types"
-version = "0.5.0"
+version = "0.5.1"
```

Comment:
So the policy for dependent crates is to bump the the patch when updating a dependency?

Reply:
I don't think we have a hard policy on that but I think it's a good practise if applied sensibly? Do you see any downsides?

Reply:
Seems fine to me - was just curious what our convention was

Reply:
I think that depends on whether the change is breaking, I don't think it is of ethereum-types and primitive-types, but please correct me if I'm wrong.

Reply:
Or if you mean in general, I don't think we should do this in every case, but I'd like to publish new version of primitive-types as well.

Reply:
Yes of course my original comment is poorly phrased. Just when I've updated a dependency like this in the past (non breaking minor version) I was unsure what was the correct bump for the parent crate.